pages/getting_started/explore.rst

```diff
@@ -0,0 +1,106 @@
+
+.. For more details, please see :ref:`views` 
+
+
+System
+^^^^^^
+
+Overview
+""""""""
+
+The Overview page displays information relating to the administration of the Graylog instance. It contains information on system notifications, system job status, ingestion rates, Elasticsearch cluster health, indexer failures, Time configuration and the system event messages.
+
+
+Configuration
+"""""""""""""
+
+The Configuration page allows users to set options or variables related to searches, message processors and plugins.
+
+Nodes
+"""""
+
+The Nodes page contains summary status information for each Graylog node. Detailed health information and metrics are available from buttons displayed on this page.
+
+Inputs
+""""""
+
+Usually the first thing configured after initial system setup, Inputs are used to tell Graylog on which port to listen or how to go and retrieve event logs. The Inputs page allows users to create and configure new inputs, to manage extractors, to start and stop inputs, get metrics for each input and to add static fields to incoming messages. 
+
+Outputs
+"""""""
+
+Outputs are used to define methods of forwarding data to remote systems, including port, protocol and any other required information. Out of the box, Graylog supports STDOUT and GELF outputs, but users may write their own and more are available in the :ref:`marketplace`.
+
+Authentication
+""""""""""""""
+
+The Authentication page is used to configure Graylog's authentication providers and manage the active users of this Graylog cluster. Graylog supports LDAP or Active Directory for both authentication and authorization.
+
+Content Packs
+"""""""""""""
+
+Content packs accelerate the set-up process for a specific data source. A content pack can include inputs/extractors, streams, dashboards, alerts, saved searches and pipeline processors. 
+
+Any program element created within Graylog may be exported as Content Packs for use on other systems. These may be kept private by the author, for use in quick deployment of new nodes internally, or may be shared with the community via the Graylog Marketplace. For example, users may create custom Inputs, Streams, Dashboards, and Alerts to support a security use case. These elements may be exported in a content pack and then imported on a newly  installed Graylog instance to save configuration time and effort. 
+
+Users may download content packs created and shared by other users via the :ref:`marketplace`. User created content packs are not supported by Graylog, but instead by their authors.
+
+.. List of Supported Elements
```

Comment:
Is content here missing? Did you want to add something more? Shall we kick this?